### ISSUE TITLE
Mac window position multiple monitors

### ISSUE THREAD

**Issue by dsotirescu** [adobe/brackets#10669]
On a MacBook Pro with multiple monitors, when opening Brackets again, it does not remember the monitor was last opened on.

This persistence was working way back, now not anymore.

Release 1.1 build 1.1.0-15558

**Comment by redmunds** [comment]
@dsotirescu Are you seeing what I logged in issue #10755 ? If so, which version of Mac are you using?

**Comment by dsotirescu** [comment]
Same OS as yours, always the latest. I reproduced your issue. In my case Brackets doesn't always open on the focused monitor, 50% of the time. I have 2 external monitors, thunderbolt + 4K monitor by Dell. Running on latest Mac Book Pro.

Sent from my iPad Air 2

On Mar 29, 2015, at 12:20 PM, Randy Edmunds <notifications@github.com> wrote:

@dsotirescu Are you seeing what I logged in issue #10755 ? If so, which version of Mac are you using?
